ecrtokenrefresher/pkg/secrets/aws/aws.go

```diff
@@ -0,0 +1,185 @@
+package aws
```

Comment:
Would be good to do a git mv on this file from the previous so it just shows up as changed/keeps history

Reply:
noted.